src/app/public/modules/colorpicker/colorpicker.component.scss

```diff
@@ -25,12 +24,6 @@
   overflow: hidden;
 }
 
-.sky-colorpicker-reset-button::before {
-  content: '\f00d';
-  font-family: FontAwesome;
-  font-size: $sky-font-size-base;
-}
-
```

Comment:
☝️ This declaration was causing the CSS to not load properly in AoT builds. Now that we have the `sky-icon` component, I just replaced the CSS declaration with the component in the template file above.